changelog.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
     and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.9.771] - 2019-02-19
+### Fixed
+* Azure SQL DB support for `New-DbaLogin` [#5100](https://github.com/sqlcollaborative/dbatools/issues/5100)
```

Comment:
If you would not mind updating this to specify `SQL Login`, Azure AD is not supported in the command right now.